stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -167,7 +167,7 @@ public fun DefaultMessageContainer(
                             DeletedMessageContent()
                         } else {
                             Column {
-                                attachmentFactory?.factory?.invoke(
+                                attachmentFactory?.Content(
```

Comment:
`attachmentFactory?.Content` this looks uncomfortable, but I don't know if we can do anything to make the syntax nicer. I guess Composable functions are just not meant to be members? 😀 Are there examples of this in first-party APIs perhaps?

Reply:
Yeah this is very ugly - I generally prefer the parameter/function property approach.

I'm thinking of just going back to that because it's a bit nicer, even though this API might make more sense